src/murfey/client/__init__.py

```diff
@@ -136,6 +142,24 @@ def run():
 
     log.info("Starting Websocket connection")
 
+    start_dc = input(
+        "Press 'D' to start a new Data Collection or press any other key to continue:"
+    )
+    if start_dc == "D":
+        start_time = str(datetime.datetime.now())
+        image_directory = str(args.destination)
+        image_suffix = input("Enter the image suffix: ")
+        visit = str(args.visit)
+        dc_params = {
+            "type": "start_dc",
+            "start_time": start_time,
+            "session_id": session_id,
+            "image_directory": image_directory,
+            "image_suffix": image_suffix,
+            "visit": visit,
+        }
+        ws.send(json.dumps(dc_params))
```

Comment:
fundamentally, I think the decision when a new datacollection starts should be made by the server, not the client?